config/locales/es-ES/kaminari.yml

```diff
@@ -1,23 +1,20 @@
----
 es:
   helpers:
     page_entries_info:
       entry:
         one: entrada
         other: entradas
-        zero: entradas
       more_pages:
         display_entries: Mostrando <strong>%{first}&nbsp;-&nbsp;%{last}</strong> de un total de <strong>%{total} %{entry_name}</strong>
       one_page:
         display_entries:
           one: Hay <strong>1 %{entry_name}</strong>
           other: Hay <strong>%{count} %{entry_name}</strong>
-          zero: No se han encontrado %{entry_name}
```

Comment:
@Andrulko this is another example of `zero` that would read strangly.

Instead of "Entry %{entry_name} wasn't found" I don't know what would get the user :( But I understand it must be something configurable somehow